components/view/entity.js

```diff
@@ -0,0 +1,53 @@
+import { default as UTILS } from '/js/deer-utils.js'
+import { default as DEER } from '/js/deer-config.js'
+import DeerView from './view.js'
+
+const template = obj => {
+    let tmpl = `<h2>${UTILS.getLabel(obj)}</h2>`
+    let list = ``
+
+    for (let key in obj) {
```

Comment:
Normalization in front of this makes `obj.id` and `obj["@id"]`, and so the entity templates come out with an id and an @id field.  Not a terrible thing, can we reduce it to just one of them?

![image](https://user-images.githubusercontent.com/3287006/185474199-ce17dadc-5b77-4ecc-8e98-2ff391f33901.png)

Reply:
Having tested most of the templates, the @id + id thing happens in most templates.

Reply:
It should happen everywhere. The place to change this would be on the `DEER.SUPPRESS` array which prevents `@context` and `__rerum` from rendering